modules/helm-installing-helm.adoc

```diff
@@ -32,7 +32,7 @@ You can also find the URL to the latest binaries from the {product-title} web co
 +
 [source,terminal]
 ----
-$ helm version
+# helm version
```

Comment:
Is there a reason to change the CLI prompt here? There is no need to run this command with admin right.

Reply:
@boczkowska I did this because the earlier steps in this procedure are all run using the "#" prompt, and there weren't any steps to switch back to a normal user or anything before running this command. 

Are you okay with this change, or would you like me to revert these ones?

Reply:
It doesn't really matter if this particular command is run with admin rights or not, so I'd leave `$` 

Hmm, I don't think we need to explicitly specify how to run commands as a superuser (sudo, su etc), it's developer-facing documentation in the end, but I can add instructions obviously.

Reply:
Okay, I'll go ahead and revert it then, thanks.